packages/sdks/snippets/sveltekit/src/routes/live-preview/+page.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import type { BuilderContent } from '@builder.io/sdk-svelte';
+  import { onMount } from 'svelte';
+  import { subscribeToEditor } from '@builder.io/sdk-svelte';
+
+  export let data: {
+    content: BuilderContent | null;
+  };
+
+  let content: BuilderContent | null = data.content;
+  let isLoading = !content;
```

Comment:
this `isLoading` is redundant here since we don't actually use it for data fetching progress. I'd remove it and remove the `Loading...` message in the UI too

```suggestion
```